Stripe/PublicHeaders/STPPaymentIntentEnums.h

```diff
@@ -0,0 +1,94 @@
+//
+//  STPPaymentIntentEnums.h
+//  Stripe
+//
+//  Created by Daniel Jackson on 6/27/18.
+//  Copyright © 2018 Stripe, Inc. All rights reserved.
+//
+
+/**
+ Status types for an STPPaymentIntent
+ */
+typedef NS_ENUM(NSInteger, STPPaymentIntentStatus) {
+    /**
+     Unknown status
+     */
+    STPPaymentIntentStatusUnknown,
+
+    /**
+     This PaymentIntent requires a Source
+     */
+    STPPaymentIntentStatusRequiresSource,
+
+    /**
+     This PaymentIntent needs to be confirmed
+     */
+    STPPaymentIntentStatusRequiresConfirmation,
+
+    /**
+     The selected Source requires additional authentication steps.
+     Additional actions found via `next_source_action`
+     */
+    STPPaymentIntentStatusRequiresSourceAction,
+
+    /**
+     Stripe is processing this PaymentIntent
+     */
+    STPPaymentIntentStatusProcessing,
+
+    /**
+     The payment has succeeded
+     */
+    STPPaymentIntentStatusSucceeded,
+
+    /**
+     Indicates the payment must be captured, for STPPaymentIntentCaptureMethodManual
```

Comment:
"Indicates the payment must be captured. The `captureMethod` is also set to `STPPaymentIntentCaptureMethodManual`."?

Reply:
I guess what you have is fine, I'm just wondering what's the intention behind having a separate `captureMethod` field and if we can communicate that reason.